docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md

```diff
@@ -3,60 +3,51 @@ title: Using Visual Studio Tools for Docker (Visual Studio on Windows)
 description: Containerized Docker Application Lifecycle with Microsoft Platform and Tools
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 09/22/2017
+ms.date: 09/12/2018
 ---
 # Using Visual Studio Tools for Docker (Visual Studio on Windows)
 
-The developer workflow when using Visual Studio Tools for Docker is similar to the workflow when using Visual Studio Code and Docker CLI (in fact, it is based on the same Docker CLI), but it is easier to get started, simplifies the process, and provides greater productivity for the build, run, and compose tasks. It's also able to execute and debug your containers via simple actions like F5 and Ctrl+F5 from Visual Studio. Even more, with Visual Studio 2017, in addition to being able to run and debug a single container, you also can run and debug a group of containers (a whole solution) at the same time if they are defined in the same docker-compose.yml file at the solution level.
+The developer workflow when using Visual Studio Tools for Docker is similar to the workflow when using Visual Studio Code and Docker CLI (in fact, it is based on the same Docker CLI), but it is easier to get started, simplifies the process, and provides greater productivity for the build, run, and compose tasks. It's also able to execute and debug your containers via simple actions like F5 and Ctrl+F5 from Visual Studio. With the optional Container Orchestration support, in addition to being able to run and debug a single container, you also can run and debug a group of containers (a whole solution) at the same time if they are defined in the same *docker-compose.yml* file at the solution level.
```

Comment:
- Split this first sentence into 2 to improve machine translation.
- developer workflow --> development workflow
- it is --> it's
- Container Orchestration --> container orchestration
- you also can run --> you can run
- they are --> they're
- Recommend splitting the last sentence into 2 to improve machine translation.